webdriver/tests/delete_cookie/META.yml

```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1044449
```

Comment:
delete.py is mentioned, but the bug isn't about this failure, so seems not worth linking.